packages/gguf/src/types.ts

```diff
@@ -138,5 +138,5 @@ export interface GGUFTensorInfo {
 export interface GGUFParseOutput {
 	metadata: GGUFMetadata;
 	tensorInfos: GGUFTensorInfo[];
-	paramCount?: number;
+	parameterCount?: number;
```

Comment:
this comment applies https://github.com/huggingface/huggingface.js/pull/637#discussion_r1581189423

Reply:
yes

Reply:
`GGUFParseOutput ` stays as it is.

The return of the function should look like: `GGUFParseOutput  & { parameterCount?: number }`

Reply:
ok i'll tweak this 🙏

Reply:
322af6bab0de2099df3f622688e8e22df07176a5